gui/main.py

```diff
@@ -348,6 +355,21 @@ def __init__(self, *args, **kwargs):
     def do_startup(self):
         Gtk.Application.do_startup(self)
 
+        quit_action = Gio.SimpleAction.new("quit", None)
+        quit_action.connect("activate", self.on_quit)
+        self.add_action(quit_action)
+
+        self.set_accels_for_action("app.quit", ["<Primary>Q"])
+
+        self.set_accels_for_action("win.clone", ["<Primary>D"])
```

Comment:
i would suggest using enum for menu instead of using strings (ie the same but without quotes), that will make easier for static analysis tools to catch typos

also it will be great to write shortcut in a button tooltip or start using GtkShortcutsShortcut for new gtk shortcut shortcut help concept